app/workers/janitor_worker.rb

```diff
@@ -7,7 +7,9 @@ def perform
     return unless ThreeScale.config.janitor_worker_enabled
 
     PurgeOldUserSessionsWorker.perform_async
-    PurgeStaleObjectsWorker.perform_later(EventStore::Event.name, DeletedObject.name)
+    PurgeStaleObjectsWorker.perform_later("EventStore::Event", "DeletedObject")
+    DeleteAllStaleObjectsWorker.perform_later("MessageRecipient", "Message")
```

Comment:
Why not?

```suggestion
    PurgeStaleObjectsWorker.perform_later("EventStore::Event", "DeletedObject", "MessageRecipient", "Message")
```

Reply:
Purge stale worker creates a background job for each individual object. And we have millions of stale recipients and probably messages. I'm not sure we wont overload our sidekiq redis. I think it is more efficient to just DELETE the db rows.

I would rather in the future move `EventStore::Event` and `DeletedObject` to use `DeleteAllStaleObjectsWorker`. But I'd rather do in another PR when we see this one working good first.

Reply:
You convinced me.